Orange/widgets/evaluate/owconfusionmatrix.py

```diff
@@ -426,11 +426,13 @@ def sum_item(value, border=""):
     def send_report(self):
         if self.results is not None and self.selected_learner:
             index = self.selected_learner[0]
+            cmatrix = confusion_matrix(self.results, index)
+            n= len(cmatrix)
```

Comment:
Depending on what `self.tablemodel` is, the cost of computing the number of rows can be O(1) operation.